### PR TITLE
Ziad/mint2.0 page

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/MintPage/index.tsx [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import PausedMint from "./PausedMint"
+import MintV2Page from "./MintV2Page"
 
-export default PausedMint
```
Comment (sweetmantech) [inline components/MintPage/index.tsx:3]: are we using `<PausedMint />` anymore?

can you delete the component if it's unused?

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

File: components/MintPage/MintV2Page.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+  const mintNFT = async () => {
+    if (!mintQuantity) return
+
+    setIsMintLoading(true)
+    const response = await mint(mintQuantity)
+    if (!response.err) {
+      setIsMintLoading(false)
+      setOpenSuccessModal(true)
+      return
```
Comment (crazzywizard) [inline components/MintPage/MintV2Page.tsx]: why do we need a return here? we could remove `setIsMintLoading(false)` and the return correct?

File: components/MintPage/MintV2Page.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+          </div>
+          <div className="flex justify-center gap-x-[20px] pt-[30px] fade_in_text">
+            <MintCTAButton
+              id="open_sea"
+              bgLink="/assets/MintV2/opensea.svg"
+              blurBgLink="/assets/MintV2/opensea.png"
+              link="https://opensea.io/collection/cre8orsaipeps"
+            />
+            <MintCTAButton
+              id="open_sea"
```
Comment (crazzywizard) [inline components/MintPage/MintV2Page.tsx]: id should be different....

File: components/MintPage/MintV2Page.tsx (hunk 2 above)
Comment (towardsnewtech) [reply components/MintPage/MintV2Page.tsx]: removed duplicated code 🚮

File: components/MintPage/index.tsx (hunk 1 above)
Comment (towardsnewtech) [reply components/MintPage/index.tsx:3]: removed this component 🚮

File: hooks/mintDay/useCre8orMintV2.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import { ethers } from "ethers"
+
+const useCre8orMintV2 = () => {
+  const signer = useEthersSigner({ chainId: process.env.NEXT_PUBLIC_TESTNET ? 5 : 1 })
+
+  const mint = async (quantity) => {
+    try {
+      const contract = new Contract(process.env.NEXT_PUBLIC_CRE8ORS_ADDRESS, cre8orAbi, signer)
+
+      const value = ethers.utils.parseUnits(Number(0.05 * quantity).toString(), 18)
```
Comment (sweetmantech) [inline hooks/mintDay/useCre8orMintV2.tsx]: should not be hard-coding the `value`

expected:
- call `process.env.NEXT_PUBLIC_CRE8ORS_ADDRESS.salesConfig().publicSalePrice to get msg.value`

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

File: pages/_app.tsx [hunk 5]
```diff
@@ -43,15 +44,17 @@ function MyApp({ Component, pageProps }: AppProps) {
...
-            <SessionProvider>
-              <AdminProvider>
-                <ChatProvider>
-                  <Component {...pageProps} />
-                  <ToastContainer />
-                  <Analytics />
-                </ChatProvider>
-              </AdminProvider>
-            </SessionProvider>
+            <MintProvider>
```
Comment (sweetmantech) [inline pages/_app.tsx:47]: MintProvider can be limited to the `/mint` route right?

do we need to wrap the base `_app`?

File: pages/mint/index.tsx [hunk 6]
```diff
@@ -1,16 +1,6 @@
+import { NextPage } from "next"
 import MintPage from "../../components/MintPage"
-import SeoHead from "../../components/SeoHead"
-import { MintProvider } from "../../providers/MintProvider"
 
-const Mint = () => (
-  <MintProvider>
```
Comment (sweetmantech) [inline pages/mint/index.tsx:6]: why did you remove the `<MintProvider />` from here to the core `_app`?

File: pages/profile/[address].tsx [hunk 7]
```diff
@@ -49,10 +48,6 @@ export const getServerSideProps: GetServerSideProps<ProfileProps> = async ({ par
   }
 }
 
-const Profile = () => (
-  <ProfileProvider>
```
Comment (sweetmantech) [inline pages/profile/[address].tsx:53]: why are we moving these providers?

**Review by SweetmanTech — COMMENTED** [review]
why are we moving `Providers` to core `_app.tsx`?

what was wrong with `/route` specific `providers`?

**Review by SweetmanTech — APPROVED** [review]
